distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -17,12 +16,20 @@ import project.Errors.Common.Not_Found
 import project.Errors.Common.Type_Error
 import project.Error.Error
 import project.Errors.Illegal_Argument.Illegal_Argument
+import project.Errors.Problem_Behavior.Problem_Behavior
 import project.Function.Function
 import project.Math
+import project.Meta
 import project.Nothing.Nothing
 import project.Panic.Panic
 import project.Random
+import project.Warning.Warning
 
+import project.IO
+
+# We have to import also conversion methods, therefore, we import all from the Ordering
+# module
```

Comment:
```suggestion
## We have to import also conversion methods, therefore, we import all from the Ordering
   module
```

I think the convention is to use `##` for the multiline comments. Unless there was some issue with it?

Reply:
Btw. I think this raises a good point that we may want to have a way to import conversion methods _without_ necessarily importing `all`. What do you think? Shall we start a discussion?

I guess one way to do so should be to do `import project.Data.Ordering` instead. I'm pretty sure the not-`from` imports _do_ import the extension methods within.

Reply:
But maybe a more explicit way for this would be good too. Just a random thought, low priority. Definitely independent of this PR.

Reply:
> Shall we start a discussion?

I don't know. I guess it depends on whether the current behavior is the expected one, i.e., if I import a type with `import Standard.Base.Data.Ordering.Comparable.Comparable`, then all its extension methods from that module are imported as well, but not any conversion methods - neither those with `Comparable` as target, neither those with `Comparable` as source. Is this intuitive/expected/bug?

Reply:
Honestly, I don't know.

My intuition would be that if I import a type, probably it's extension methods are imported too... But probably only its and not others.

I'd appreciate a way to import _only_ extension methods from a file, but that's just my subjective view. Also definitely not something of high priority, as we are doing fine with current state of the import system, I think.